inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -61,7 +61,7 @@ <h3 class="text-center header-text">
 
       <holding-pen-dashboard-item
         template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"
-        filter_string="source=arXiv&workflow_name=HEP&is-update=false"
+        filter_string="source=arXiv&workflow_name=HEP&q=!_extra_data.is-update:true"
```

Comment:
`q=!is-update:true` seems to work also.

Reply:
I think not, it seems that this query returns you the whole set of results.

Reply:
you're right! I didn't look close enough :+1: